angular-hometask/src/app/modules/courses-page/entities/directives/course-plate-border.directive.ts

```diff
@@ -0,0 +1,23 @@
+import { Directive, ElementRef, Input, OnChanges } from '@angular/core';
+
+@Directive({
+  selector: '[appCoursePlateBorder]'
+})
+export class CoursePlateBorderDirective implements OnChanges {
+
+  @Input('appCoursePlateBorder') creationDate: Date;
+
+  constructor(public el: ElementRef) {
+  }
+
+  ngOnChanges() {
```

Comment:
if you are using ngOnChanges pls add change: SimpleChanges, and check if you have updating in your input data

Reply:
6d5882784ead660a1ae8d72c3b651d7e3da42a32